### DISCUSSION TITLE
How to auto download latest chapters?

### DISCUSSION THREAD

**Discussion by Sujan-13** [lncrawl/lightnovel-crawler#2077] [Q&A]
I am currently reading Shadow Slave, however it is an ongoing novel and I have caught upto the latest bulk chapter, since only 1-2 new chapters are uploaded each day, it is simply tediou to repeat the process of typing the url again and again. Is there a way to automate the process and if so how? Thanks

**Comment by HongYue1** [comment]
make a script with python, you can use an external file to keep count of the latest chapter and use systemd/Task Scheduler to run it every day automatically.

**Reply by monkeeay** [reply]
What would that script look like
